internal/k8shandler/deployment.go

```diff
@@ -40,9 +45,13 @@ func NewDeployment(deploymentName string, namespace string, loggingComponent str
 				ObjectMeta: metav1.ObjectMeta{
 					Name: deploymentName,
 					Labels: map[string]string{
-						"provider":      "openshift",
-						"component":     component,
-						"logging-infra": loggingComponent,
+						"provider":                     "openshift",
+						"component":                    component,
+						"logging-infra":                loggingComponent,
+						"app.kubernetes.io/name":       loggingComponent,
+						"app.kubernetes.io/component":  constants.CollectorName,
+						"app.kubernetes.io/created-by": constants.ClusterLoggingOperator,
+						"app.kubernetes.io/managed-by": constants.ClusterLoggingOperator,
```

Comment:
These are the same labels as on the `Deployment`. Can be replaced by a variable.